nzedb/Binaries.php

```diff
@@ -1058,6 +1057,8 @@ protected function storeHeaders(array $headers, $multiGroup)
 	 * @param array $returnArray
 	 * @param array $headers
 	 * @param int   $msgCount
+	 *
+	 * @return array
```

Comment:
You can also add the return type to the method signature.

Reply:
php 5.6 does not support them, hence i will not be adding them untill we drop it.

Reply:
It IS dropped